lib/components/post-editor-sidebar-component.js

```diff
@@ -90,7 +90,7 @@ export default class PostEditorSidebarComponent extends BaseContainer {
 	}
 
 	publicizeToTwitterAccountDisplayed() {
-		const twitterAccountSelector = By.xpath( `//h5[text()='Twitter']/..//label` );
+		const twitterAccountSelector = By.css( 'div.editor-sharing__publicize-connection > label > span' );
```

Comment:
Thanks for the help! With it I was able to fix this issue, which included adding a data attribute to Calypso. I created a branch for it, too, which you can see here: https://github.com/Automattic/wp-calypso/tree/adding-data-attributes-for-wpe2e-tests